src/core/public/chrome/constants.ts

```diff
@@ -31,3 +31,11 @@
 export const OPENSEARCH_DASHBOARDS_ASK_OPENSEARCH_LINK = 'https://forum.opensearch.org/';
 export const GITHUB_CREATE_ISSUE_LINK =
   'https://github.com/opensearch-project/OpenSearch-Dashboards/issues/new/choose';
+
+export const WORKSPACE_APP_ID = 'workspace';
```

Comment:
I'm thinking to avoid redefining `WORKSPACE_APP_ID` here as it's the same as the one we defined in workspace plugin, we should not maintain another copy of plugin id in OSD core.

One quick solution:
```typescript
// src/plugins/workspace/public/plugin.ts -> setup()
core.workspaces.client.setAppId(WORKSPACE_APP_ID)

// src/core/public/workspace/workspaces_client.ts
public setAppId(id: string) {
  this.appId = id;
}

// src/core/public/chrome/chrome_service.tsx
  application.getUrlForApp(workspaces.client.appId, {
    path: '/',
    absolute: true,
  }),
```

@wanglam @SuZhou-Joe what's your opinion?

Reply:
I'd prefer to use a simple `import` from `core/public/workspace/constant.ts`, because it's a static variable and should not have anything to do with the runtime `workspace_client`.